### PR TITLE
Content for search elastic v2

### PR DESCRIPTION
**Pull request by EParzefall**
Adding information on version 2 of the Full Text Search app coming soon. Text is awaiting approval by engineering and product management. Do not merge before the release.
This will fix issue #3996 and issue #3929.

### CODE REVIEW

**Comment by EParzefall** [comment]
@micbar @pmaier1 I've removed the simple query sting links and the explanation how to use wildcards.

**Review by pmaier1 — APPROVED** [review]
Thanks!

**Comment by EParzefall** [comment]
Minimal update for now to reflect the changes described in
https://github.com/owncloud/search_elastic/pull/212#issuecomment-983361975

**Comment by mmattel** [comment]
There is no limitation to core (10.9)

Backport to 10.9, 10.8 and 10.7

File: modules/user_manual/pages/files/webgui/search.adoc [hunk 1]
```diff
@@ -7,14 +7,15 @@ ownCloud comes with a regular search function allowing you to find files by thei
...
+Things get tricky when you don't remember what the file you're looking for was called and guess work doesn't help. A key word search within text documents is only possible if the Full Text Search app is installed. Contact your admin if you need it and it's not installed. With this app enabled, you can search within files of the most common formats. Like with the regular search, you can enter only the first part of a string and it matches all occurances of words starting with the search string.
+
+If you are looking for all terms containing a specific string like in the above example "pain", wildcards can be used, e.g. an asterisk: *pain gives you results like braine, Spain and painful. Using a question mark instead of the asterisk limits the preceeding characters to exactly one. In this example, only Spain would show up if you entered the search string "?pain".
```
Comment (phil-davis) [inline modules/user_manual/pages/files/webgui/search.adoc:20]: @EParzefall @mmattel why does `*pain` match `braine` ?

**Pull request merged**